client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/style.scss

```diff
@@ -40,25 +42,18 @@ $font-family: "SF Pro Text", $sans;
 		box-sizing: border-box;
 		margin-top: 50px;
 		margin-bottom: 32px;
-	}
-
-	.pattern-assembler__preview {
-		flex-basis: 100%;
-		margin-inline-start: 32px;
-		background: #f6f6f6;
-		box-shadow: 0 0 2px rgba(0, 0, 0, 10%);
-		border-radius: 4px;
-		display: flex;
-		align-items: center;
-		justify-content: center;
-		margin-bottom: 32px;
-		margin-top: -25px;
+		position: relative;
+		z-index: 0;
 	}
 
 	.pattern-layout {
 		display: flex;
 		flex-direction: column;
 		height: 100%;
+		width: 100%;
+		background: #fdfdfd;
```

Comment:
One small suggestion I have here is to, as a rule of thumb, try to find if the color is already defined in `@automattic/calypso-color-schemes` since Stepper imports it [here](https://github.com/Automattic/wp-calypso/blob/trunk/client/landing/stepper/declarative-flow/internals/global.scss#L5).

The benefit of this is that we ensure that the color is part of the "color system", and that we could apply different color schemes in an easier way.

Reply:
This is very helpful. Thanks for sharing it! I found the blue color for the buttons, but the other color is hardcoded [here](https://github.com/Automattic/wp-calypso/blob/trunk/client/landing/stepper/declarative-flow/internals/global.scss#L14).